src/components/common/YourCompany/OfficeAddresses.vue

```diff
@@ -471,19 +471,19 @@ export default class OfficeAddresses extends Mixins(CommonMixin) {
   private recDeliveryAddress = {} as AddressIF
 
   // The 4 validation events from each BaseAddress sub-component:
-  private mailingAddressValid: boolean = true
-  private deliveryAddressValid: boolean = true
-  private recMailingAddressValid: boolean = true
-  private recDeliveryAddressValid: boolean = true
+  private mailingAddressValid = true
+  private deliveryAddressValid = true
+  private recMailingAddressValid = true
+  private recDeliveryAddressValid = true
```

Comment:
These (above and below) are equivalent. In fact, some linters or TS plugins will complain that it's redundant to declare the variable as boolean and also assign a boolean value. So I simplified this.